dev-support/hbase-personality.sh

```diff
@@ -500,7 +500,7 @@ function shadedjars_rebuild
 
   local -a maven_args=('clean' 'verify' '-fae' '--batch-mode'
     '-pl' 'hbase-shaded/hbase-shaded-check-invariants' '-am'
-    '-Dtest=NoUnitTests' '-DHBasePatchProcess' '-Prelease'
+    '-DskipTests' '-DHBasePatchProcess' '-Prelease'
```

Comment:
I believe the `NoUnitTests` is intentional in that it forces the `test-compile` stage to happen even without running any tests. `-DskipTests` will bypass the whole `test` phase and so the unit test code is never compiled.

Reply:
No, -DskipTests will executing the test-compile stage.

https://maven.apache.org/surefire/maven-surefire-plugin/examples/skipping-tests.html

> If you absolutely must, you can also use the maven.test.skip property to skip compiling the tests. maven.test.skip is honored by Surefire, Failsafe and the Compiler Plugin.

The '-DskipTests' is only honored by surefire plugin so it can not skip a stage which is mainly controlled by other plugins...